ddl/ddl_api.go

```diff
@@ -3046,6 +3046,13 @@ func SetDirectResourceGroupUnit(resourceGroupSettings *model.ResourceGroupSettin
 		resourceGroupSettings.IOReadBandwidth = stringVal
 	case ast.ResourceUnitIOWriteBandwidth:
 		resourceGroupSettings.IOWriteBandwidth = stringVal
+	case ast.ResourceBurstableOpiton:
+		limit := int64(0)
+		if boolValue {
+			// negative means no limit within burst setting.
+			limit = -1
```

Comment:
What's the different between -1 and 0?

Reply:
I add some comments, it's not by convention:)